### ISSUE TITLE
Allow sending a key multiple times

### ISSUE THREAD

**Issue by dit7ya** [iberianpig/fusuma-plugin-sendkey#6]
For example, my VolumeUP and VolumeDown keys have a very small step, so it would be great if I could just write something like `      sendkey: VOLUMEUP * 5` to send 5 keystrokes at back to back.

**Comment by iberianpig** [comment]
Please consider `command:` or set threshold lower
```
swipe:
  3:
    left:
      command: "fusuma-sendkey VOLUMEDOWN && fusuma-sendkey VOLUMEDOWN && fusuma-sendkey VOLUMEDOWN"
```

**Comment by iberianpig** [comment]
fixed #34
